e2e-tests/tests/features/dbbackup_psmdb/10-assert.yaml

```diff
@@ -32,6 +32,12 @@ kind: PerconaServerMongoDB
 metadata:
   name: test-psmdb-cluster
 spec:
+  backup:
+    configuration:
+      backupOptions:
+        oplogSpanMin: 0
```

Comment:
Why is this value 0?

Reply:
the `.backup.oplogSpanMin` is set to 0 as a side effect, because we need to have the field since we configure the backup options object for PBM. However the 0 value [is ignored by pbm](https://github.com/percona/percona-backup-mongodb/blob/2a6b65bbe29bb4dadadfd9d84ceeea27aced26bd/pbm/config/config.go#L170), so this `oplogSpanMin` does not introduce any changes.